modules/ppcp-settings/src/SettingsModule.php

```diff
@@ -439,12 +438,15 @@ function ( array $payment_methods ) use ( $container ): array {
 				$general_settings = $container->get( 'settings.data.general' );
 				assert( $general_settings instanceof GeneralSettings );
 
-				$merchant_data    = $general_settings->get_merchant_data();
-				$merchant_country = $merchant_data->merchant_country;
+				$messages_apply = $container->get( 'button.helper.messages-apply' );
+				assert( $messages_apply instanceof MessagesApply );
+				$pay_later_eligible = $messages_apply->for_country();
```

Comment:
`$messages_apply` and `$pay_later_eligible` seems never used, check it and remove if not needed.